gprofiler/metadata/application_identifiers.py

```diff
@@ -104,6 +104,16 @@ def gunicorn_to_app_id(self, wsgi_app_spec: str, process: Process) -> str:
         wsgi_app_file = wsgi_app_spec.split(":", maxsplit=1)[0]
         return f"gunicorn: {wsgi_app_spec} ({_append_python_module_to_proc_wd(process, wsgi_app_file)})"
 
+    def gunicorn_get_app_name(self, cmdline_list: List[str]) -> str:
+        # method improving appid selection:
+        # https://github.com/Granulate/gprofiler/issues/704
+        cmdline_colon_list = list(filter(lambda x: ":" in x, cmdline_list))
+        for index, arg in enumerate(cmdline_list):
+            for colon_arg in cmdline_colon_list:
+                if arg == colon_arg and not cmdline_list[index - 1].startswith("-"):
+                    return arg
```

Comment:
Why do you require `cmdline_colon_list`? Can't you replace `arg == colon_arg` with `":" in arg`?